candy.css

```diff
@@ -3,6 +3,24 @@ body {
   font-family: "Nunito", sans-serif;
 }
 
+h2 {
+  color: #ea1c2c;
+  font-weight: 100;
+  font-size: 2.5rem;
+}
+
+
+p {
+  color: #f498b8;
+  font-weight: 100;
+  font-size: 1.125rem;
+  line-height: 2
+}
+
+.content {
+  margin-top: 100px;
```

Comment:
moving forward we will be using rems (look up the difference between px, em, and rem)

Reply:
no correction for now